src/dbnode/persist/fs/write.go

```diff
@@ -98,6 +98,7 @@ type indexEntries []indexEntry
 func (e indexEntries) releaseRefs() {
 	for i := range e {
 		e[i].id = nil
+		e[i].tags = ident.Tags{}
```

Comment:
Can you just use an empty index entry instead of this?

```
var empty indexEntry
for i := range e {
  e[i] = empty
}
```

This way you get the memclr optimization too

Reply:
Just for me to understand, is this equivalent to `the` memset optimizations?

Reply:
Yep

Reply:
Yeah fair call